R/TaskRegr_dti.R

```diff
@@ -0,0 +1,34 @@
+#' @title Diffusion Tensor Imaging (DTI) Regression Task
+#'
+#' @name mlr_tasks_dti
+#' @format [R6::R6Class] inheriting from [TaskRegr].
+#'
+#' @description
+#' This dataset contains two functional covariates and three scalar covariate. The goal is
+#' to predict the PASAT score.
+#'
+#' This is a subset of the full dataset, which is contained in the package `refund`.
+#'
+#' @section Construction:
+#'
+#' ```
+#' mlr_tasks$get("dti")
+#' tsk("dti")
+#' ```
+#' @references `r format_bib("goldsmith2011penalized") `
+#'
+NULL
+
+load_task_fuel = function(id = "dti") {
+  b = as_data_backend(load_dataset("dti", package = "mlr3fda"))
+
+  task = TaskRegr$new(
```

Comment:
nicht alle spalten sind features. Du musst die `col_role` `"features"` setzen

Reply:
was sollen sonst sein als `feature`?  Für mich macht es Sinn die drin zu haben `case Numeric vector of multiple sclerosis case status: 0 - healthy control, 1 - MS case;` weil man ja dann ein Indikator dafür hat ob multiple sclerosis existiert oder nicht, aber ich nehme es gerne raus, wenn es nicht sinnvoll ist

Reply:
Ok ja man kann es auch drin lassen, dann lernt mal eben auch den Effekt von MS auf den PASAT Score.